h/static/scripts/group-forms/components/test/CreateEditGroupForm-test.js

```diff
@@ -163,16 +178,21 @@ describe('CreateGroupForm', () => {
     });
   });
 
-  it("doesn't show an error message initially", async () => {
-    const { wrapper } = createWrapper();
+  it('displays a create-new-group form', async () => {
+    const { wrapper, elements } = createWrapper();
+    const headerEl = elements.header.element;
+    const nameEl = elements.name.fieldEl;
+    const descriptionEl = elements.description.fieldEl;
+    const submitButtonEl = elements.submitButton.element;
 
+    assert.equal(headerEl.text(), 'Create a new private group');
+    assert.equal(nameEl.getDOMNode().value, '');
+    assert.equal(descriptionEl.getDOMNode().value, '');
+    assert.equal(submitButtonEl.text(), 'Create group');
+    assert.isFalse(wrapper.exists('[data-testid="back-link"]'));
     assert.isFalse(wrapper.exists('[data-testid="error-message"]'));
-  });
-
-  it("doesn't show a loading state initially", async () => {
-    const { wrapper } = createWrapper();
-
     await assertInLoadingState(wrapper, false);
+    assert.isFalse(wrapper.exists('[data-testid="check"]'));
```

Comment:
I often find it helpful to extract even these small `wrapper.exists`-type expressions into a helper (eg. `checkIsVisible(wrapper)`) to make it easier to update the selector if for any reason it needs to change in future.

Reply:
Done